src/libs/type-utils/lodash-fp-helpers.test.ts

```diff
@@ -0,0 +1,451 @@
+import _ from 'lodash/fp'
+import { delay } from 'src/libs/utils'
+import { SafeCurry2, SafeCurry3 } from 'types/lodash-fp/lodash-fp.utils'
+
+import { AnyFn, AnyPromiseFn, GenericFn, GenericPromiseFn } from './general-types'
```

Comment:
All existing Terra UI code uses absolute imports.

Reply:
hmm... I poked around a bit on the contributing wiki but couldn't find this callout.... can you point me to it?

I am curious on the reasoning, because I have found that relative paths for 'in same directory or in a sub-directory of focused module area' is very handy in that it makes the code more portable without refactoring import paths.  For things not nearby, yes absolute paths is great in that it calls that out.

As an aside, when we consider shared components/packages, we will need to do extra work to allow for absolute pathing  in those consumable packages so that the consuming packages don't run into problems with the package exports.

Reply:
To my surprise, I also did not find it documented anywhere. 

> I am curious on the reasoning, because I have found that relative paths for 'in same directory or in a sub-directory of focused module area' is very handy in that it makes the code more portable without refactoring import paths. For things not nearby, yes absolute paths is great in that it calls that out.

Yeah, this makes sense to me. I have no objection to relative imports, I only wanted to flag that this was a break with convention in existing code.